### ISSUE TITLE
Failing travis

### ISSUE THREAD

**Issue by ankurankan** [pgmpy/pgmpy#852]
Travis is failing on our current dev: https://travis-ci.org/pgmpy/pgmpy/jobs/201953353:

```python
======================================================================
ERROR: test_find_resonable_stepsize (pgmpy.tests.test_sampling.test_continuous_sampling.TestHMCInference)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/pgmpy/pgmpy/pgmpy/tests/test_sampling/test_continuous_sampling.py", line 41, in test_find_resonable_stepsize
    stepsize = self.hmc_sampler._find_reasonable_stepsize(np.array([-1, 1, -1]))
  File "/home/travis/build/pgmpy/pgmpy/pgmpy/sampling/HMC.py", line 128, in _find_reasonable_stepsize
    condition = (acceptance_prob ** a) > (2 ** (-a))
ValueError: Integers to negative integer powers are not allowed.

======================================================================
ERROR: test_sample (pgmpy.tests.test_sampling.test_continuous_sampling.TestHMCInference)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/pgmpy/pgmpy/pgmpy/tests/test_sampling/test_continuous_sampling.py", line 55, in test_sample
    num_samples=10000, trajectory_length=4)
  File "/home/travis/build/pgmpy/pgmpy/pgmpy/sampling/HMC.py", line 474, in sample
    stepsize = self._find_reasonable_stepsize(initial_pos)
  File "/home/travis/build/pgmpy/pgmpy/pgmpy/sampling/HMC.py", line 128, in _find_reasonable_stepsize
    condition = (acceptance_prob ** a) > (2 ** (-a))
ValueError: Integers to negative integer powers are not allowed.

======================================================================
ERROR: test_sampling (pgmpy.tests.test_sampling.test_continuous_sampling.TestNUTSInference)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/pgmpy/pgmpy/pgmpy/tests/test_sampling/test_continuous_sampling.py", line 127, in test_sampling
    return_type='recarray')
  File "/home/travis/build/pgmpy/pgmpy/pgmpy/sampling/NUTS.py", line 527, in sample
    stepsize = self._find_reasonable_stepsize(initial_pos)
  File "/home/travis/build/pgmpy/pgmpy/pgmpy/sampling/HMC.py", line 128, in _find_reasonable_stepsize
    condition = (acceptance_prob ** a) > (2 ** (-a))
ValueError: Integers to negative integer powers are not allowed.
```

@khalibartan Can you please look into this ?

**Comment by khalibartan** [comment]
@ankurankan I'm a bit busy because of my mid-terms right now. Can you fix this?
It is not that difficult. The error is caused by backward incompatible changes made in numpy1.12 and changes in power `**` operator . I was not able to reproduce this on my personal system. Just put in a if else condition for checking whether a is 1 or -1 and when -1 just convert the expression to 1/(expression).

FootNotes:
https://github.com/numpy/numpy/releases/tag/v1.12.0rc1

**Comment by ankurankan** [comment]
Temporarily fixed: 1cda0fd030ddb27a1be5654466b32a47a5e1fe92
